packages/styles/components/Tooltip.module.scss

```diff
@@ -0,0 +1,104 @@
+@import '~@venice/styles/core';
+
+.tooltip {
+  align-items: center;
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+
+  .tooltipContent {
+    border: 1px solid #e5e5e5;
+    border-radius: var(--border-radius);
+    display: none;
+    min-width: 150px;
+    padding: 10px var(--space-sm);
+    position: absolute;
+    transition: opacity 0.5s;
+    z-index: 1;
+
+    &::after {
+      border-style: solid;
+      border-width: 5px;
```

Comment:
We should create a variable (in SCSS) to border-width, for example

```
$arrow-size: 5px
```

In top of file

Reply:
Done ✅ 
10d2eb6